build.gradle

```diff
@@ -33,6 +35,23 @@ buildScan {
     licenseAgree = 'yes'
 }
 
+	task cleanShared() {
+		group 'Configuration'
+		description 'Removes ALL shared libraries from the servlet container.'
+		def libs = serverbase+"/shared/lib"
+		FileTree tree = fileTree(dir: libs)
+		delete fileTree(dir:libs , include: '*')
+	}
+
+	 task cleanTomcat() {
+		group 'Configuration'
+		description 'Removes the deployed uPortal from the servlet container.'
+		def uPortal = serverbase+"/webapps/uPortal"
+		printf uPortal
```

Comment:
This is not left over I want to print what directory it is deleting when gralde task is executed